sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/_tests_/e2e/student-scholastic-standings.institutions.controller.saveScholasticStanding.e2e-spec.ts

```diff
@@ -0,0 +1,321 @@
+    const endpoint = `/institutions/scholastic-standing/location/${collegeFLocation.id}/application/${application.id}`;
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .post(endpoint)
+      .send(payload)
+      .auth(institutionUserToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.UNPROCESSABLE_ENTITY)
+      .expect({
+        message:
+          "Cannot report a change for application with status other than completed.",
+        errorType: INVALID_OPERATION_IN_THE_CURRENT_STATUS,
+      });
+  });
+
+  it("Should create a new scholastic standing when the institution user requests it.", async () => {
+    // Arrange
+    const application = await saveFakeApplicationDisbursements(
+      db.dataSource,
+      {
+        institutionLocation: collegeFLocation,
+      },
+      {
+        applicationStatus: ApplicationStatus.Completed,
+        currentAssessmentInitialValues: {
+          assessmentWorkflowId: "some fake id",
+          assessmentDate: addDays(1),
+          studentAssessmentStatus: StudentAssessmentStatus.Completed,
+        },
+      },
+    );
+    // Createa student appeal for the application and its student assessment
+    const studentAppeal = await createFakeStudentAppeal({
+      application: application,
+      studentAssessment: application.currentAssessment,
+    });
+    application.currentAssessment.studentAppeal = studentAppeal;
+    await db.application.save(application);
+    const payload: ScholasticStandingAPIInDTO = {
+      data: {
+        booksAndSupplies: 1000,
+        dateOfChange: getISODateOnlyString(new Date()),
+        scholasticStandingChangeType:
+          StudentScholasticStandingChangeType.ChangeInIntensity,
+      },
+    };
+    // Mock the form service to validate the dry-run submission result.
+    // TODO: Form service must be hosted for E2E tests to validate dry run submission
+    // and this mock must be removed.
+    const formService = await getProviderInstanceForModule(
+      appModule,
+      AppInstitutionsModule,
+      FormService,
+    );
+    const dryRunSubmissionMock = jest.fn().mockResolvedValue({
+      valid: true,
+      formName: SCHOLASTIC_STANDING_FORM_NAME,
+      data: { data: payload.data },
+    });
+    formService.dryRunSubmission = dryRunSubmissionMock;
+    const endpoint = `/institutions/scholastic-standing/location/${collegeFLocation.id}/application/${application.id}`;
+
+    // Act/Assert
+    let createdScholasticStanding: number;
+    await request(app.getHttpServer())
+      .post(endpoint)
+      .send(payload)
+      .auth(institutionUserToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.CREATED)
+      .expect((response) => {
+        console.log("response.body: ", response.body);
+        expect(response.body.id).toBeGreaterThan(0);
+        createdScholasticStanding = +response.body.id;
+      });
+    const newScholasticStanding = await db.studentScholasticStanding.findOne({
```

Comment:
I would suggest to have the query performed using the application repository and asserting the scholastic standing through the application's current assessment.